karoo_gp/test/test_population.py

```diff
@@ -82,10 +82,12 @@ def test_population_class(default_kwargs, default_evaluate_params,
     dataset_params = load_data(kernel, save_dir='test')
 
     # Initialize population using dataset terminals
-    kwargs = dict(default_kwargs)
-    kwargs['tree_pop_max'] = 10
-    kwargs['terminals'] = dataset_params['terminals']
-    kwargs['fitness_type'] = dataset_params['fitness_type']
+    kwargs = {
+        **default_kwargs,
+        'tree_pop_max': 10,
+        'terminals': dataset_params['terminals'],
+        'fitness_type': dataset_params['fitness_type']
```

Comment:
```suggestion
        'fitness_type': dataset_params['fitness_type'],
```
When you have a list/dict of elements, with one element per line, always remember to add a comma after the last element too.  If you do, it's easier to add new elements afterwards since you don't have to remember to add the comma the line before.  In addition the diff looks nicer, since you only add one line instead of adding one and editing the previous one to add the comma.